docs/README.md

```diff
@@ -21,6 +21,7 @@ The benefit to using Semantic Conventions is in following a common naming scheme
 Semantic Conventions are defined for the following areas:
 
 * **[General](general/README.md): General Semantic Conventions**.
+* [CICD](cicd/cicd-metrics.md): Semantic Conventions for CICD systems.
```

Comment:
Should we add the following line to `CODEOWNERS`?
` /docs/cicd/ @open-telemetry/specs-semconv-approvers @open-telemetry/semconv-cicd-approvers`

Reply:
Done in 344eeedbeda8a77ef258cc305947cb48edc4e4ea